src/web-map.js

```diff
@@ -409,6 +423,24 @@ export class WebMap extends HTMLMapElement {
               {target: this}}));
```

Comment:
@prushforth A lot of the other code in web-map.js is specific to mapml-viewer and most likely does not work on web-map. We will need to go over this in the future. Ex:
`document.activeElement.nodeName === "MAPML-VIEWER"`

Reply:
Whoops! definitely, put it on the backlog...